package.json

```diff
@@ -83,21 +83,23 @@
   "devDependencies": {
     "@babel/core": "^7.11.1",
     "@mdn/dinocons": "^0.0.7",
-    "@mdn/minimalist": "^0.0.3",
-    "@storybook/addon-a11y": "^6.0.20",
-    "@storybook/addon-actions": "^6.0.20",
-    "@storybook/addon-essentials": "^6.0.20",
-    "@storybook/addon-links": "^6.0.20",
-    "@storybook/node-logger": "^6.0.20",
+    "@mdn/minimalist": "^0.0.4",
+    "@storybook/addon-a11y": "^6.0.21",
+    "@storybook/addon-actions": "^6.0.21",
+    "@storybook/addon-essentials": "^6.0.21",
+    "@storybook/addon-links": "^6.0.21",
+    "@storybook/node-logger": "^6.0.21",
     "@storybook/preset-create-react-app": "^3.1.4",
-    "@storybook/react": "^6.0.20",
+    "@storybook/react": "^6.0.21",
+    "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^10.4.9",
     "@types/jest": "^26.0.10",
     "@types/react": "^16.9.48",
     "@types/react-dom": "^16.9.8",
     "@types/react-modal": "^3.10.6",
     "babel-loader": "^8.1.0",
     "cheerio": "1.0.0-rc.3",
+    "eslint-plugin-jest-dom": "^3.2.2",
```

Comment:
If this is needed you should perhaps consider adding it to `master` and that would explain why it's needed.

Reply:
Because of the decision re: unit testing, I am going to remove this.